### PR TITLE
Update MSIDLABS B2C authority in E2E tests

### PR DESCRIPTION
**Pull request by derisen**
B2C is deprecating `login.microsoftonline.com` endpoint for Azure AD B2C tenants, so MSIDLABS has moved on to `b2clogin.com `endpoints, which caused tests to break. This PR updates the authorities to `b2clogin.com`

### CODE REVIEW

File: samples/msal-node-samples/auth-code/config/B2C-AAD.json [hunk 1]
```diff
@@ -2,8 +2,8 @@
     "authOptions":
         {
             "clientId": "e3b9ad76-9763-4827-b088-80c7a7888f79",
-            "authority": "https://login.microsoftonline.com/tfp/msidlabb2c.onmicrosoft.com/B2C_1_SISOPolicy",
-            "knownAuthorities": ["login.microsoftonline.com"]
+            "authority": "https://msidlabb2c.b2clogin.com/tfp/msidlabb2c.onmicrosoft.com/B2C_1_SISOPolicy",
```
Comment (tnorling) [inline samples/msal-node-samples/auth-code/config/B2C-AAD.json]: Is tfp still required with this change?

Comment (derisen) [reply samples/msal-node-samples/auth-code/config/B2C-AAD.json]: Aah, probably not : ) let me remove it

Comment (derisen) [reply samples/msal-node-samples/auth-code/config/B2C-AAD.json]: done

**Review by tnorling — APPROVED** [review]
Do any of the brower/react/angular tests need to be updated as well?

**Comment by derisen** [comment]
> Do any of the brower/react/angular tests need to be updated as well?

@tnorling 🤦‍♂️ yes -and a unit test apparently. Updated all

File: change/@azure-msal-common-24ae5dc6-8dbc-4616-ad73-14c6869154d1.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Update B2C authority format in tests #5375",
+  "packageName": "@azure/msal-common",
+  "email": "dogan.erisen@gmail.com",
```
Comment (Robbie-Microsoft) [inline change/@azure-msal-common-24ae5dc6-8dbc-4616-ad73-14c6869154d1.json]: should this be your Microsoft email?

**Review by Robbie-Microsoft — APPROVED** [review]

File: change/@azure-msal-common-24ae5dc6-8dbc-4616-ad73-14c6869154d1.json (hunk 2 above)
Comment (derisen) [reply change/@azure-msal-common-24ae5dc6-8dbc-4616-ad73-14c6869154d1.json]: sorry -fixed!

**Pull request merged**